packages/playground/src/weblets/tf_jitsi.vue

```diff
@@ -0,0 +1,215 @@
+          validators.minLength('Name must be at least 2 characters.', 2),
+          validators.maxLength('Name cannot exceed 50 characters.', 50),
+        ]"
+        #="{ props }"
+      >
+        <input-tooltip tooltip="Instance name.">
+          <v-text-field label="Name" v-model="name" v-bind="props" />
+        </input-tooltip>
+      </input-validator>
+
+      <SelectSolutionFlavor
+        v-model="solution"
+        :small="{ cpu: 2, memory: 4, disk: 100 }"
+        :medium="{ cpu: 4, memory: 16, disk: 500 }"
+        :large="{ cpu: 8, memory: 32, disk: 1000 }"
+      />
+
+      <Networks v-model:ipv4="ipv4" v-model:planetary="planetary" v-model:mycelium="mycelium" v-model:ipv6="ipv6" />
+
+      <input-tooltip inline tooltip="Click to know more about dedicated machines." :href="manual.dedicated_machines">
+        <v-switch color="primary" inset label="Dedicated" v-model="dedicated" hide-details />
+      </input-tooltip>
+
+      <input-tooltip inline tooltip="Renting capacity on certified nodes is charged 25% extra.">
+        <v-switch color="primary" inset label="Certified" v-model="certified" hide-details />
+      </input-tooltip>
+
+      <TfSelectionDetails
+        :filters="{
+          ipv4,
+          ipv6,
+          certified,
+          dedicated,
+          cpu: solution?.cpu,
+          solutionDisk: solution?.disk,
+          memory: solution?.memory,
+          rootFilesystemSize,
+        }"
+        require-domain
+        v-model="selectionDetails"
+      />
+
+      <manage-ssh-deployemnt @selected-keys="updateSSHkeyEnv($event)" />
+    </d-tabs>
+
+    <template #footer-actions="{ validateBeforeDeploy }">
+      <v-btn color="secondary" @click="validateBeforeDeploy(deploy)" text="Deploy" />
+    </template>
+  </weblet-layout>
+</template>
+
+<script lang="ts" setup>
+import { calculateRootFileSystem, type GridClient } from "@threefold/grid_client";
+import { computed, type Ref, ref } from "vue";
+
+import { manual } from "@/utils/manual";
+
+import { useLayout } from "../components/weblet_layout.vue";
+import { useGrid, useProfileManager } from "../stores";
+import type { Flist, solutionFlavor as SolutionFlavor } from "../types";
+import { ProjectName } from "../types";
+import { deployVM } from "../utils/deploy_vm";
+import { deployGatewayName, getSubdomain, rollbackDeployment } from "../utils/gateway";
+import { normalizeError } from "../utils/helpers";
+import { generateName } from "../utils/strings";
+
+const layout = useLayout();
+const profileManager = useProfileManager();
+
+const name = ref(generateName({ prefix: "jt" }));
+const solution = ref() as Ref<SolutionFlavor>;
+const flist: Flist = {
+  value: "https://hub.grid.tf/tf-official-apps/jitsi-latest.flist",
+  entryPoint: "/sbin/zinit init",
+};
+const dedicated = ref(false);
+const certified = ref(false);
+const ipv4 = ref(false);
+const ipv6 = ref(false);
+const mycelium = ref(true);
+const planetary = ref(true);
```

Comment:
Based on https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3287 we should enable only the `mycelium` network

Reply:
Done, only mycelium is enabled by default